II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,13 @@
+function getAverage(numbers) {
+  var result = numbers.filter(member => typeof member === "number");
+  return findAverage(result);
+}
+
+function findAverage(arr) {
+  return arr.reduce((a, b) => a + b, 0) / arr.length;
+}
```

Comment:
This is great! Not only you used `reduce`, but you also extracted it to its own function.
This makes the code so pleasant to read :smiley: